corehq/apps/sso/backends.py

```diff
@@ -81,6 +81,13 @@ def authenticate(self, request, username, idp_slug, is_handshake_successful):
             user, web_user = self._create_new_user(request, username, async_signup)
             is_new_user = True
 
+        if not is_new_user and not web_user.is_active:
+            web_user.is_active = True
+            web_user.save()
+            request.sso_new_user_messages['success'].append(
+                _("User account for {} reactivated.").format(web_user.username)
```

Comment:
"Account for {} has been re-activated."

Reply:
Thank you! Fixed here: dda30afd918255289f41037dd4454ccdd97b12ed